index.html

```diff
@@ -0,0 +1,211 @@
+          </p>
+          <h2 class="hero-date">2022.06.06(Mon)~11(Sat)</h2>
+          <span class="hero-address"
+            >@ Taj Pamodzi Hotel</span
+          >
+        </div>
+      </section>
+      <section id="the-training">
+        <div class="training-section desktop-padding">
+          <h2 class="the-training-title the-underline"> Events </h2>
+          <div class="events">
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/women.svg"
+                alt="apparels"
+              />
+              <h2>Shopping</h2>
+              <p>Buy fabulous apparels from top designers in the country and abroad.</p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/runway.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Runway</h2>
+              <p>
+                Watch as top models walk the Runway
+                 in extraodinary brands.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/workshop.svg"
+                alt="Lecture Talk"
+              />
+              <h2>workshop</h2>
+              <p>
+                Learn fashion and designing from the experts.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/forum.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Forum</h2>
+              <p>
+                Have the time to share your opinions and your thoughts with
+                expert designers and fashion moguls.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/exhibitions.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Exhibitions</h2>
+              <p>
+                Showcase your brand to thousands of fashion Ethusiasts and get exlusive access to 
+                the main event. 
+              </p>
+            </div>
+          </div>
+          <button type="button" class="register-btn">
+            Register For the event
+          </button>
+          <a href="#" class="see-all-program">See the Whole Program</a>
+        </div>
+      </section>
+      <section id="trainers">
+        <div class="desktop-padding">
+          <h2 class="trainers-section-title the-underline">Designers</h2>
+          <div id="trainers-container"></div>
+          <button id="more-trainers" class="more-trainers">
+            More <img src="images/icons/down.svg" alt="" />
+          </button>
```

Comment:
- [ ] Hi please add an appropriate alt text to this `img` tag. this is good for accessibility reasons.